packages/cli/src/models/network/NetworkController.ts

```diff
@@ -210,10 +210,9 @@ export default class NetworkController {
       await this._setSolidityLibs(contract);
       log.info(`Uploading ${contract.schema.contractName} contract as ${contractAlias}`);
       const contractInstance = await this.project.setImplementation(contract, contractAlias);
+      const { types, storage } = contract.schema.storageInfo || { types: null, storage: null };
```

Comment:
If there's no difference between having `type` and `storage` assigned as `undefined` instead of `null`, you can simply change this to:
```suggestion
      const { types, storage } = contract.schema.storageInfo;
```

Reply:
That was my first try, but actually, the ts compiler complained if I did that